src/cable.cr

```diff
@@ -5,7 +5,7 @@ require "./cable/**"
 
 # TODO: Write documentation for `Cable`
 module Cable
-  VERSION = "0.1.0"
+  VERSION = "0.1.1"
```

Comment:
Let's hold off on this in the PR. We can handle doing the version bump

Reply:
no probs, @jwoertink I'm thinking maybe you could make a release tag for 0.1.0 just now on master, then we can merge in a bunch of changes and at the end bump the release?